defold-rive/pluginsrc/plugin.cpp

```diff
@@ -658,167 +670,128 @@ static void GenerateAABB(RiveFile* file)
     ++v;
 }
 
-static inline void Mat2DToMat4(const rive::Mat2D m2, dmVMath::Matrix4& m4)
+// *******************************************************************************************************
+
+// Used when processing the Rive draw events, to produce draw calls
+struct RiveEventsDrawcallContext
 {
-    m4[0][0] = m2[0];
-    m4[0][1] = m2[1];
-    m4[0][2] = 0.0;
-    m4[0][3] = 0.0;
-
-    m4[1][0] = m2[2];
-    m4[1][1] = m2[3];
-    m4[1][2] = 0.0;
-    m4[1][3] = 0.0;
-
-    m4[2][0] = 0.0;
-    m4[2][1] = 0.0;
-    m4[2][2] = 1.0;
-    m4[2][3] = 0.0;
-
-    m4[3][0] = m2[4];
-    m4[3][1] = m2[5];
-    m4[3][2] = 0.0;
-    m4[3][3] = 1.0;
-}
+    dmRive::RiveVertex*     m_VertexBuffer;
+    int*                    m_IndexBuffer;
+    dmRive::RenderObject*   m_RenderObjects;
+    //dmRiveDDF::RiveModelDesc::BlendMode m_BlendMode;
```

Comment:
Not used?

Reply:
Not yet anyways.
I left it as a to-do for the editor